apache/zeppelin
# ZEPPELIN-2527 Changed editor cursor to thin

### What is this PR for?
Previously, it's hard to recognize the cursor since it was too thick. After this PR, it will show the cursor same as other input boxes for consistency.

See the attached GIFs for comparison.


### What type of PR is it?
[Improvement]

### Todos

### What is the Jira issue?
* [ZEPPELIN-2527]

### How should this be tested?

1. Build: mvn clean package -Denforcer.skip -DskipTests -Drat.skip
2. Open a paragraph
3. Check the cursor

### Screenshots (if appropriate)
Before the change
![cursor-change-pr-before](https://user-images.githubusercontent.com/1881135/27826498-dd7c27e6-60d0-11e7-887d-f7abf7b2acd8.gif)


After the change
![cursor-change-pr](https://user-images.githubusercontent.com/1881135/27826466-b4fb8028-60d0-11e7-986f-81904a4cb2e6.gif)


### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?
